pkg/clients/elasticloadbalancing/elb/elb.go

```diff
@@ -0,0 +1,223 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elb
+
+import (
+	"context"
+	"encoding/json"
+	"sort"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/awserr"
+	elb "github.com/aws/aws-sdk-go-v2/service/elasticloadbalancing"
+	"github.com/aws/aws-sdk-go-v2/service/elasticloadbalancing/elasticloadbalancingiface"
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/crossplane/provider-aws/apis/elasticloadbalancing/v1alpha1"
+	clients "github.com/crossplane/provider-aws/pkg/clients"
+)
+
+const (
+	// ELBNotFound is the code returned by AWS when there is not ELB with a specified name.
+	ELBNotFound = "LoadBalancerNotFound"
+)
+
+// A Client handles CRUD operations for Elastic Load Balancing resources.
+type Client elasticloadbalancingiface.ClientAPI
```

Comment:
Rather than this, can we create an `interface`, with all the functions used in it, and then initiate that interface type?
Generally, I saw this practice, any reason to choose otherwise?

Reply:
I noticed there was already an interface in the AWS SDK and found it a bit easy to use than to create an interface. 
Doing this does expose all of the methods in `elasticloadbalancingiface.ClientAPI` to ELB controller which I don't see as a problem.

Reply:
Seems good.